src/middlewared/middlewared/plugins/network.py

```diff
@@ -644,8 +644,12 @@ async def __check_failover_disabled(self):
             return
         if await self.middleware.call('failover.status') == 'SINGLE':
             return
-        if not (await self.middleware.call('failover.config'))['disabled']:
-            raise CallError('Disable failover before performing interfaces changes.')
```

Comment:
Shouldnt failover.status be returning SINGLE if there is no VIP and HA is not configured yet? I wonder what it is returning and if thats not the real regression

Reply:
No, not in this scenario. The system is being configured with an HA license and also has a zpool configured meaning it won't return `SINGLE`.

Reply:
So why does this work in 11.3?

Reply:
```
        interfaces = await self.middleware.call('interface.query')
        if not any(filter(lambda x: x['state']['carp_config'], interfaces)):
            return 'SINGLE'
```

Reply:
It should be SINGLE if there is no carp configured

Reply:
Not sure if this doesn't work on 11.3. I'll investigate there.